### PR TITLE
Clickable links

### PR DESCRIPTION
**Pull request by rb50**
Makes website and email links on individual business pages clickable, and sorts listings alphabetically regardless of how they appear on the google sheet

### CODE REVIEW

**Review by Johanna-hub — APPROVED** [review]
Looks great!

**Comment by Johanna-hub** [comment]
The regex makes my brain hurt but you did a fab job!

File: components/app/BusinessInfo.jsx [hunk 1]
```diff
@@ -89,84 +91,117 @@ const BusinessLink = styled.a`
...
+
+
 const Tag = ({ type, ...props }) => (
-  <Box bg="rgba(103, 128, 159, 1)" py={1} px={16} borderRadius={4} {...props}>
-    <TagText color="#fff">
-      {type}
-    </TagText>
-  </Box>
+    <Box bg="rgba(103, 128, 159, 1)" py={1} px={16} borderRadius={4} {...props}>
+        <TagText color="#fff">
```
Comment (macintoshhelper) [inline components/app/BusinessInfo.jsx:117]: Hi. Seems that an editor config for 4 spaces has slipped into this PR. Maybe worth deciding on a project wide `.editorconfig` for tab width, etc. There is a `.prettierrc` config for tab width, I'm not a big fan of prettier though, especially if it isn't being run in a build step or CI. I'll open an issue about it

Comment (rb50) [reply components/app/BusinessInfo.jsx:117]: Yes I just realised this when I did the PR - yes it'll be good to have a consistent config!

**Pull request merged**